common/src/main/java/com/thoughtworks/go/server/service/result/OperationResult.java

```diff
@@ -43,6 +43,8 @@
 
     ServerHealthState unauthorized(String message, String description, HealthStateType id);
```

Comment:
From my understanding `unauthorized` was always used instead of `forbidden`. I think it's probably safe to rename this message to `forbidden` instead. By extension, the `forbidden` method you added below should not be needed.

@jyotisingh WDYT?

Reply:
Yes, makes sense. 
If an unauthenticated user tries to access a url which doesn't allow anonymous access, I believe `BasicAuthenticationWithChallengeFilter` would return 401 so the request would never reach the controller, so I think we are covered for that too.

Reply:
> From my understanding unauthorized was always used instead of forbidden. I think it's probably safe to rename this message to forbidden instead. By extension, the forbidden method you added below should not be needed.

I will delete `unauthorized ` method